omod/src/main/resources/liquibase.xml

```diff
@@ -195,5 +195,31 @@
 							 tableName="metadatamapping_metadata_source"
 							 columnNames="name" />
 	</changeSet>
-
+	
+	<changeSet id="metadatamapping-2016-01-06-0800" author="jasonvena">
+		<preConditions onFail="MARK_RAN">
+			<columnExists tableName="metadatamapping_metadata_term_mapping"
+						  columnName="metadata_class" />
+		</preConditions>
```

Comment:
Well, I said it myself: it's a good idea to follow established practices. I appreciate the fact that you have checked out what others have done. However, in this case I find the established practice a bit problematic. Let's see what others think: https://talk.openmrs.org/t/liquibase-precondition-conventions/4282